test/test_suite_sqlalchemy.py

```diff
@@ -1,28 +1,115 @@
 from sqlalchemy import __version__ as sa_version
-from sqlalchemy.testing import combinations
 from sqlalchemy.testing import skip
 from sqlalchemy.testing.suite import *  # noqa
 from sqlalchemy.testing.suite import ComponentReflectionTest as _ComponentReflectionTest
+from sqlalchemy.testing.suite import ComponentReflectionTestExtra as _ComponentReflectionTestExtra
+from sqlalchemy.testing.suite import CompositeKeyReflectionTest as _CompositeKeyReflectionTest
+from sqlalchemy.testing.suite import HasIndexTest as _HasIndexTest
+from sqlalchemy.testing.suite import HasTableTest as _HasTableTest
 from sqlalchemy.testing.suite import ExpandingBoundInTest as _ExpandingBoundInTest
 from sqlalchemy.testing.suite import LongNameBlowoutTest as _LongNameBlowoutTest
 from sqlalchemy.testing.suite import NumericTest as _NumericTest
+from sqlalchemy.testing.suite import QuotedNameArgumentTest as _QuotedNameArgumentTest
+from sqlalchemy.testing.suite import TrueDivTest as _TrueDivTest
 from sqlalchemy.testing.suite import UnicodeSchemaTest as _UnicodeSchemaTest
 
 
 class ComponentReflectionTest(_ComponentReflectionTest):
+    @testing.combinations(
+        (False,), (True, testing.requires.schemas), argnames="use_schema"
+    )
+    @testing.requires.foreign_key_constraint_reflection
+    def test_get_foreign_keys(self, connection, use_schema):
+        if connection.dialect.driver != "asyncpg":
+            super().test_get_foreign_keys(connection, use_schema, None)
+
     def test_get_indexes(self, connection):
-        if connection.dialect._is_v202plus and sa_version >= "1.4":
+        if (
+            connection.dialect.driver != "asyncpg"
+            and connection.dialect._is_v202plus
+            and sa_version >= "1.4"
```

Comment:
does this `sa_version` check actually work? i don't think a string comparison seems right here..